src/oc/list.cpp

```diff
@@ -195,11 +195,13 @@ Item* lappendvoid(List* list, void* obj) {
     return insertvoid(list, obj);
 }
 
-void hoc_l_delete(Item* item) {
+Item* hoc_l_delete(Item* item) {
```

Comment:
Is this a (doubly) linked list or something? Does this return the next item in the chain then (and more importantly, why, instead of using `void`)? If there's good reason for returning from it, I think it'd be good to document the behavior.

Reply:
I can remove this unrelated change.